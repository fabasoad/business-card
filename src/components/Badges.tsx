import * as React from 'react'
import * as imgDigitalOceanBadge from '../assets/images/badges/digital-ocean-forest-2021.png'
import SectionTitle from './Controls/SectionTitle'
import { Col, Container, Row } from 'react-bootstrap'
import { useTranslation } from 'react-i18next'

export default function Badges() {
  const { t } = useTranslation()
  return (
    <div id="badges" className="light-component">
      <SectionTitle>{t('business-card-badges-title')}</SectionTitle>
      <Container>
        <Row>
          <Col sm md lg className="text-center">
            <a href="https://tree-nation.com/profile/digitalocean" rel="noopener noreferrer" target="_blank">
              <img className="mb-4 mx-4" src={imgDigitalOceanBadge} alt="Digital Ocean Forest Badge" title="DigitalOcean Forest 2021" />
            </a>
          </Col>
          {/* <Col sm md lg className="text-center">
            <Iframe
              url={'https://www.worldcommunitygrid.org/getDynamicImage.do?memberName=fabasoad&mnOn=true&stat=3&imageNum=1&rankOn=false&projectsOn=false&special=false&link=1&memberId=1103124'}
              frameBorder={0}
              name="di"
              scrolling="no"
              display="inline"
            />
          </Col> */}
        </Row>
      </Container>
    </div>
  )
}
