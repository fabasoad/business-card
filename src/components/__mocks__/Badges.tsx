import * as React from 'react'

export default function Badges() {
  return (
    <div data-testid="Badges"></div>
  )
}
