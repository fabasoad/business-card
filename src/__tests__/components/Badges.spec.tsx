/// <reference types="jest" />
import * as React from 'react'
import { render } from '@testing-library/react'
import Badges from '../../components/Badges'

test('should render Badges correctly', () => {
  const { container } = render(<Badges/>)
  expect(container.firstChild).toMatchSnapshot()
})
