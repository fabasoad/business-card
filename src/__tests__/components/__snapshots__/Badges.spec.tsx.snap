// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Badges correctly 1`] = `
<div
  class="light-component"
  id="badges"
>
  <div
    class="section-title text-center center"
  >
    <h2>
      business-card-badges-title
    </h2>
    <hr />
  </div>
  <div
    class="container"
  >
    <div
      class="row"
    >
      <div
        class="col-sm col-md col-lg text-center"
      >
        <a
          href="https://tree-nation.com/profile/digitalocean"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            alt="Digital Ocean Forest Badge"
            class="mb-4 mx-4"
            src="test-file-stub"
            title="DigitalOcean Forest 2021"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
